packages/leonardo-data-client/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@terra-ui-packages/leonardo-data-client",
```

Maybe... ?

```suggestion
  "name": "@terra-ui-packages/data-client-leonardo",
```

So all the data client packages end up grouped together.

---

actually, I was suggesting we go with all of the data-clients be a sub-folder deeper.... /packages/data-clients/data-client-core, etc.

which would make the naming "grouping" a non-issue... I think leonardo-data-client is a nicer name.